app/assess/models/assessor_task_list.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class _SubCriteria:
```

Comment:
I imagine this would be separated out as the dataclass will be used elsewhere. IMO okay to leave here for this PR though and refactor as needed